tests/Test/AbstractFixerTestCase.php

```diff
@@ -464,6 +465,11 @@ protected function lintSource(string $source): ?string
     protected static function assertCorrectCasing(string $haystack, string $needle, string $fixerName, string $descriptionType): void
     {
         $exceptions = [
+            'PHPDoc' => [
+                'option:comment_type' => [
+                    'align_multiline_comment' => 2,
+                ],
+            ],
```

Comment:
I dropped it on local machine to check how the error will look alike if someone would use `phpdocs_` option in other fixer.

Outcome is confusing:
```
1) PhpCsFixer\Tests\Fixer\Phpdoc\AlignMultilineCommentFixerTest::testXxxFixerConfigurationDefinitions
[align_multiline_comment] `PHPDoc` must be in correct casing in option:comment_type.
Failed asserting that 1 is identical to 3.
```

Reply:
What is confusing in "`PHPDoc` must be in correct casing in option:comment_type."? It is quite clear to me what is wrong.

Reply:
it was confusing as test was complaining about plural/singular and test failure description was complaining about casing.

not the case anymore with newest state of this PR